mm2src/coins/utxo/rpc_clients.rs

```diff
@@ -2605,6 +2605,9 @@ async fn electrum_process_json(
                         },
                     };
 
+                    // TODO: Use the streaming manager instead to send the notification.
+                    // This is disabled (set as None from the provider) for now untill the refactor in #1966
+                    // is available. Leaving it as is for now to avoid lots of conflicts.
                     if let Some(sender) = scripthash_notification_sender {
```

Comment:
I think sending notification inside this fn is not the best practice: looks like a 'side effect' problem. Maybe move it into a dedicated fn

Reply:
me too. but i haven't though about where should it go yet. will try to figure that out after the refactor in 1966 lands to save me some brain power xD.